doc/tests/empty.rst

```diff
@@ -1,11 +1,19 @@
 ``empty``
 =========
 
-``empty`` checks if a variable is empty:
+``empty`` checks if a variable is
+ - an empty string
+ - an empty array
+ - an empty hash
+ - exactly ``false``
+ - exactly ``null``
```

Comment:
Last small change is to put everything back on one line to keep things consistent with the other parts of the docs: ``empty`` checks if a variable is an empty string, an empty array, an empty hash, ...